pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -132,29 +135,137 @@ masked.select(rf_no_data_cells('blue_masked'), rf_tile_sum('mask')).show(10)
 
 It's also nice to view a sample.
 
-```python show_masked
+```python
 sample = masked.orderBy(-rf_no_data_cells('blue_masked')).select(rf_tile('blue_masked'), rf_tile('scl')).first()
 display(sample[0])
 ```
 
 And the original SCL data.
 
-```python show_scl
+```python
 display(sample[1])
 ```
 
-## NoData in Arithmetic Operations
+## Changing a Tile's NoData Values
+
+One way to mask a tile is to make a new tile with a user defined NoData value. We will explore this method below. First, lets create a rasterframe from a tile with values of 0, 1, 2, and 3.
+
+```python create_dummy_tile
+tile_size = 100
+x = np.zeros((tile_size, tile_size), dtype='int16')
+
+for i in range(4):
+    x[:, i*tile_size//4:(i+1)*tile_size//4] = i
+x = Tile(x)
+
+rf = spark.createDataFrame(pd.DataFrame(data = 
+        {'tile': [x]}))
+x
+```
+
+First, we mask the value of 1 by making a new column with the user defined cell type 'uint16ud1'. Then, we mask out the value of two by making a tile with the cell type 'uint16ud2'.
+
+```python
+def get_nodata_ct(nd_val):
+    return CellType('uint16').with_no_data_value(nd_val).cell_type_name
+
+masked_rf = rf.withColumn('tile_nd_1', 
+                           rf_convert_cell_type('tile', get_nodata_ct(1))) \
```

Comment:
@vpipkt We should fix this to accept the `CellType` class directly, and not have the user do `cell_type_name`.

Reply:
Yeah that was a little annoying